network_models/xray_flux_forecast/starting_point/network.py

```diff
@@ -18,11 +18,16 @@
 # Optimizer used in training
 from keras.optimizers import adam
 
+# Uncomment to force training to take place on the CPU
+#import os
```

Comment:
There is a config parameter for this that will make the experiment.py target the CPU. You can remove this.

Reply:
removed